static/app/utils/theme/theme.chonk.tsx

```diff
@@ -81,6 +81,11 @@ const formTheme: FormTheme = {
       borderRadius: '4px',
     },
   },
+  formSpacing: {
+    md: '8px',
+    sm: '6px',
+    xs: '4px',
```

Comment:
note: I added this because we often need this 8/6/4 spacing that is adaptive to the size. I’m not super happy with having this here though